tests/Common/Reader/ReaderWithFilter/BaseReaderWithLikeTestCase.php

```diff
@@ -13,16 +13,20 @@ abstract class BaseReaderWithLikeTestCase extends BaseReaderTestCase
     public static function dataWithReader(): array
     {
         return [
-            'case matches' => ['email', 'seed@', [2]],
-            'case does not match' => ['email', 'SEED@', [2]],
-            'wildcard is not supported' => ['email', '%st', []],
+            'case matches, starts with search string, case sensitive: null' => ['email', 'seed@', null, [2]],
+            'case matches, ends with search string, case sensitive: null' => ['email', '@beat', null, [2]],
+            'case matches, contains search string, case sensitive: null' => ['email', 'ed@be', null, [2]],
+            'case matches, contains search string, case sensitive: true' => ['email', 'ed@be', true, [2]],
+            'case matches, contains search string, case sensitive: false' => ['email', 'ed@be', false, [2]],
+            'case does not match, contains search string, case sensitive: null' => ['email', 'SEED@', null, [2]],
```

Comment:
```suggestion
            'case matches, contains search string, case sensitive: null' => ['email', 'SEED@', null, [2]],
```

Also need tests when case sensitice is true and case does not match.